tools/admin/requests/view-request.ts

```diff
@@ -35,7 +39,7 @@ export type GetApplicationResponse = {
       | 'appNumber'
       | 'appHolepunched'
       | 'walletCardCreated'
-      | 'invoiceNumber'
+      | 'invoice'
```

Comment:
We should pick the selected invoice fields (ie. do `& { invoice: Pick<Invoice, ...> }`)